src/Microsoft.AspNetCore.Authentication.Facebook/FacebookExtensions.cs

```diff
@@ -16,6 +16,9 @@ public static AuthenticationBuilder AddFacebook(this AuthenticationBuilder build
             => builder.AddFacebook(FacebookDefaults.AuthenticationScheme, configureOptions);
 
         public static AuthenticationBuilder AddFacebook(this AuthenticationBuilder builder, string authenticationScheme, Action<FacebookOptions> configureOptions)
-            => builder.AddOAuth<FacebookOptions, FacebookHandler>(authenticationScheme, configureOptions);
+            => builder.AddFacebook(authenticationScheme, authenticationScheme, configureOptions);
```

Comment:
> You didn't actually answer the question though, would you really want duplicates in your login list?

I'd prefer that over using my custom scheme as the display name :smile: 
To avoid that, I could simply use the longest overload.

My point is that you can use the `builder.AddFacebook("custom-scheme", options => { })` overload without necessarily having multiple Facebook handlers. In this case, it wouldn't make much sense to use `custom-scheme` as the display name.